docs/Antora/modules/background_info/pages/readers.adoc

```diff
@@ -59,6 +62,7 @@ Type2 b = (Type2) a;
 
 As the Reader receives the xref:background_info:signals.adoc[Signal] as a parameter it must first establish a xref:background_info:data_path.adoc#connection[Connection] to it.
 To perform this, the Readers usually create an internal xref:glossary:glossary.adoc#input_port[Input Port] to which they connect the passed-in Signal to form a Connection and start to listen to the port events.
+A Reader can also be created with an existing input port. In this case, it won't create an internal input port but will instead use the existing one.
```

Comment:
I'd add a note that this is most often used inmodule implementations where ports are attached to the openDAQ component tree within an "Input ports" folder (likely below a Function block/Channel).